src/OrchardCore.Modules/OrchardCore.Media/Permissions.cs

```diff
@@ -31,12 +36,19 @@ public IEnumerable<PermissionStereotype> GetDefaultStereotypes()
                 new PermissionStereotype
                 {
                     Name = "Administrator",
-                    Permissions = new[] { ManageMedia, ManageAttachedMediaFieldsFolder, ManageMediaProfiles, ViewMediaOptions }
+                    Permissions = new[] {
+                        ManageMedia,
+                        ManageMediaFolder,
+                        ManageOthersMedia,
+                        ManageOwnMedia,
+                        ManageAttachedMediaFieldsFolder,
+                        ManageMediaProfiles,
+                        ViewMediaOptions }
```

Comment:
Here, the permissions added to the `Administrator` role by default, should only include the `ManageMediaFolder`, rather than the `ManageMedia`, `ManageOthersMedia`, and other related implied permissions.

This is because many of these permissions, are implied by something else, so when we add them as defaults, we should add the most senior permission.

`ManagedMediaProfiles`, and `ViewMediaOptions` look like they should remain there, as they are not implied by anything.

Reply:
ok, deleted.